data/categories.txt

```diff
@@ -6833,6 +6859,8 @@ fa:پارکینگ دوچرخه
 
 amenity-waste_basket
 en:Trash Bin|Litter Bin|Waste Basket
+en-GB:Waste Bin|Litter Bin
```

Comment:
@vng if a user has en-GB locale and types "Trash", will he find this feature?

Reply:
They'd be typing American English if they were!  But added as a synonym.